tools/docker/docker-compose-dev.yaml

```diff
@@ -57,13 +57,32 @@ services:
     volumes:
       - '../../:/app/src:z'
 
-  eda-worker:
+  default-worker:
+    image: 'localhost/aap-eda-worker'
+    build:
+      context: ../../
+      dockerfile: tools/docker/worker/Dockerfile
+      args:
+        - WORKER_CLASS: 'aap_eda.core.tasking.Default'
+    environment: *common-env
+    depends_on:
+      redis:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+    volumes:
+      - '../../:/app/src:z'
+    restart: always
+
+  activation-worker:
     deploy:
       replicas: ${EDA_NUM_WORKERS:-2}
     image: 'localhost/aap-eda-worker'
     build:
       context: ../../
       dockerfile: tools/docker/worker/Dockerfile
+      args:
+        - WORKER_CLASS: 'aap_eda.core.tasking.Activation'
```

Comment:
`WORKER_CLASS` variable should have a default value in the dockerfile, it is usually an antipattern to require in a mandatory build_arg to build an image. As this value is only consumed at execution time, not even is necessary to implement it as build_arg. It can be just an environment variable. I suggest one of the following approaches:

- Declare the envvar WORKER_CLASS with a default value in the dockerfiles, and then just override that environment variable in the docker-compose files. 

OR

- Not touch the dockerfile and just override the command line as you already do for docker-compose-stage and docker-compose-mac

The second one is the preferred one IMO, is less disruptive and would not require changes in the eda-operator